clientconn.go

```diff
@@ -921,16 +844,16 @@ type addrConn struct {
 	cancel context.CancelFunc
 
 	cc      *ClientConn
-	curAddr Address
-	addrs   []Address
+	curAddr resolver.Address
+	addrs   []resolver.Address
 	dopts   dialOptions
 	events  trace.EventLog
+	acbw    *acBalancerWrapper
```

Comment:
Should this be a SubConn type instead?  We only need it so we can notify the balancer that the ac's state changed, IIUC.